src/storage/cuckoo/cuckoo.c

```diff
@@ -296,12 +291,12 @@ cuckoo_setup(cuckoo_options_st *options, cuckoo_metrics_st *metrics)
     }
     ds = datapool_addr(pool);
 
-#if CC_ITT
-    cuckoo_malloc = __itt_heap_function_create("cuckoo_malloc", "pelikan");
+#ifdef CC_ITT
```

Comment:
the point of doing the abstraction is to avoid the `ifdefs`.

In this case I guess it makes sense, but... could we maybe move alloc to insert and call free in remove?

Reply:
Done. Need to update a cc_module a little bit see if this what you expected